src/components/CategorizationRules/CategorizationRulesTable/CategorizationRulesTable.tsx

```diff
@@ -1,5 +1,158 @@
+    }
+  }, [hasMore, fetchMore, currentPage, setCurrentPage])
+
+  const CategorizationRulesTableEmptyState = useCallback(() => {
+    return (
+      <DataState
+        status={DataStateStatus.allDone}
+        title='No rules found'
+        description='No categorization rules have been created yet. You will receive suggestions for rules to create as you categorize transactions in the bank feed.'
+        icon={<PencilRuler />}
+        spacing
+      />
+    )
+  }, [])
+
+  const CategorizationRulesTableErrorState = useCallback(() => (
+    <DataState
+      status={DataStateStatus.failed}
+      title='We couldn’t load your categorization rules'
+      description='An error occurred while loading your categorization rules. Please check your connection and try again.'
+      onRefresh={() => { void refetch() }}
+      spacing
+    />
+  ), [refetch])
+
+  const columnConfig: ColumnConfig<CategorizationRule, CategorizationRuleColumns> = useMemo(() => ({
+    [CategorizationRuleColumns.Counterparty]: {
+      id: CategorizationRuleColumns.Counterparty,
+      header: 'Counterparty',
+      cell: row => (
+        <Span ellipsis>{row.counterpartyFilter?.name}</Span>
+      ),
+    },
+    [CategorizationRuleColumns.Category]: {
+      id: CategorizationRuleColumns.Category,
+      header: 'Category',
+      cell: row => (
+        row.category
+          ? (
+            <LedgerAccountCombobox
+              label='Account'
+              value={row.category}
+              mode={CategoriesListMode.All}
+              onValueChange={() => {}}
+              isReadOnly={true}
+            />
+          )
+          : undefined
+      ),
+      isRowHeader: true,
+    },
+    [CategorizationRuleColumns.Delete]: {
+      id: CategorizationRuleColumns.Delete,
+      cell: row => (
+        <Button
+          inset
+          icon
+          onPress={() => {
+            setSelectedRule(row)
+            setShowDeletionConfirmationModal(true)
+          }}
+          aria-label='Delete rule'
+          variant='ghost'
+        >
+          <Trash2 />
+        </Button>
+      ),
+    },
+  }), [])
+
   return (
-    null
+    <Container name='CategorizationRulesTable'>
+      <PaginatedTable
+        ariaLabel='CategorizationRules'
+        data={categorizationRules}
+        isLoading={data === undefined || isLoading}
+        isError={isError}
+        columnConfig={columnConfig}
+        paginationProps={paginationProps}
+        componentName={COMPONENT_NAME}
+        slots={{
+          EmptyState: CategorizationRulesTableEmptyState,
+          ErrorState: CategorizationRulesTableErrorState,
+        }}
+      />
+      <BaseConfirmationModal
+        isOpen={showDeletionConfirmationModal}
+        onOpenChange={setShowDeletionConfirmationModal}
+        title='Delete this categorization rule?'
+        description={`Any previously auto-categorized ${selectedRule?.counterpartyFilter?.name} transactions will not be uncategorized.`}
```

Comment:
Would love to rev this copy with feedback from Justin - I think the relevant context will be something like:
```suggestion
        description={`Transactions will no longer automatically be categorized as by this rule. Any transactions previously categorized to ${selectedRule?.counterpartyFilter?.name} will not be affected.`}
```